templates/search.jsx

```diff
@@ -0,0 +1,244 @@
+var facetData = [
+  {id: 1, fieldsetLabel: "Refine By", fieldName: "refineBy", inputItems: [
+    {id: 1, value: "peer-reviewed", label: "Peer-reviewed only (##)"}
+  ]},
+  {id: 2, fieldsetLabel: "Research", fieldName: "research", inputItems: [
+    {id: 2, value: "articles", label: "Articles (##)"}, 
+    {id: 3, value: "books", label: "Books (##)"}, 
+    {id: 4, value: "theses", label: "Theses (##)"}
+  ]},
+  {id: 3, fieldsetLabel: "Content Type", fieldName: "contentType", inputItems: [
+    {id: 5, value: "video", label: "Video"}, 
+    {id: 6, value: "audio", label: "Audio"}, 
+    {id: 7, value: "images", label: "Images"}, 
+    {id: 8, value: "pdf", label: "PDF"}, 
+    {id: 9, value: "zip", label: "ZIP"}
+  ]},
+  //assumption: when display === "range", then inputItems.length === 2, inputItems[0] is start, and inputItems[1] is end
+  {id: 4, fieldsetLabel: "Publication Year", fieldName: "publicationYear", display: "range", inputItems: [
+    {id: 10, value: "start", label: "From:", fieldType: "text"}, 
+    {id: 11, value: "end", label: "To:", fieldType: "text"}
+  ]},
+  {id: 5, fieldsetLabel: "Campuses", fieldName: "campuses", inputItems: [
+    {id: 12, value: "ucb", label: "UC Berkeley"},
+    {id: 13, value: "ucd", label: "UC Davis"},
+    {id: 14, value: "uci", label: "UC Irvine"},
+  ]},
+  {id: 6, fieldsetLabel: "Departments", fieldName: "departments", display: "hierarchy", inputItems: [
+    {id: 15, value: "ahc", label: "Agricultural History Center"}, 
+    {id: 16, value: "ag-nat-resources-research-ext-centers", label: "Agriculture and Natural Resources Research and Extension Centers", inputItems: [
+      {id: 17, value: "hop-research-extension-center", label: "Hopland Research and Extension Center"},
+      {id: 18, value: "sierra-foothill-reasearch-and-extension-center", label: "Sierra Foothill Research and Extension Center"}
+    ]},
+    {id: 19, value: "american-cultures-center", label: "American Cultures Center"}
+  ]},
+  {id: 7, fieldsetLabel: "Journals", fieldName: "journals"},
+  {id: 8, fieldsetLabel: "Disciplines", fieldName: "disciplines"},
+  {id: 9, fieldsetLabel: "Reuse License", fieldName: "reuseLicense"},
+]
+
+var CurrentSearchTerms = React.createClass({
```

Comment:
How would you feel about changing to ES6 class syntax? For instance, this one would look like this:
```
class CurrentSearchTerms extends React.Component {
  render() {
```